pkg/ddl/ingest/backend.go

```diff
@@ -156,34 +142,6 @@ func (bc *litBackendCtx) CollectRemoteDuplicateRows(indexID int64, tbl table.Tab
 	return bc.handleErrorAfterCollectRemoteDuplicateRows(err, indexID, tbl, hasDupe)
 }
 
-// FinishImport imports all the key-values in engine into the storage, collects
-// the duplicate errors if any, and removes the engine from the backend context.
-// When duplicate errors are found, it will return ErrKeyExists error.
-func (bc *litBackendCtx) FinishImport(tbl table.Table) error {
-	for _, ei := range bc.engines {
-		if err := ei.ImportAndClean(); err != nil {
```

Comment:
`ImportAndClean()` is also dead code now.

Reply:
Removed in https://github.com/pingcap/tidb/pull/54149/commits/545c9e9c4bd59aa756d688ab4dae9cfdd9248d9d